### ISSUE TITLE
[AbstractBundle] Call to an undefined method NodeDefinition::children()

### ISSUE THREAD

**Issue by mmarton** [phpstan/phpstan-symfony#463]
Hi!

I would like to reopen this issue: https://github.com/phpstan/phpstan-symfony/issues/278

Extending Abstractbundle and using the configuration method there is the recommended way since symfony 6.4 as you can see here:
https://symfony.com/doc/current/bundles/configuration.html

 > There are two different ways of creating friendly configuration for a bundle:
> 1.    [Using the main bundle class](https://symfony.com/doc/current/bundles/configuration.html#bundle-friendly-config-bundle-class): this is recommended for new bundles and for bundles following the [recommended directory structure](https://symfony.com/doc/current/bundles.html#bundles-directory-structure);

in my code:
```php
use Symfony\Component\HttpKernel\Bundle\AbstractBundle;

class MyBundle extends AbstractBundle
{
    public function configure(DefinitionConfigurator $definition): void
    {
        $definition->rootNode()
            ->children()   // <- this line reports
                ->arrayNode('locales')
                    ->scalarPrototype()->end()
                ->end()
            ->end();
    }
}
```

this reports

>  Call to an undefined method Symfony\Component\Config\Definition\Builder\NodeDefinition::children().  
         🪪  method.notFound          

currently bundles just ignoring this issue, like
https://github.com/zenstruck/foundry/blob/62a1c63a000c4886f1d7222ab8bec32a502828b2/src/ZenstruckFoundryBundle.php#L51

in v2.0.9 change were added in the latest release handling something similar:
[24d8c15](https://github.com/phpstan/phpstan-symfony/commit/24d8c157aa483141b0579d705ef0aac9e1b95436) - Fix compatibility with Symfony 7.4/8.0 - fix Call to method arrayNode() on an unknown class

**Comment by mmarton** [comment]
this looks like to be fixed by
https://github.com/symfony/symfony/pull/62608
and will be released in symfony 7.4.1/8.0.1

I'll wait for the release then close this
